### PR TITLE
Updated view-group-members.md to use Get-DynamicDistributionGroupMember

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2719.

@chrisda do you know why this cmdlet is not documented? I want to confirm if I can add it here. If you approve I can also add it to the office-docs-powershell repo. Thanks!

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-dynamic-distribution-groups/view-group-members.md [hunk 1]
```diff
@@ -36,22 +36,19 @@ Dynamic distribution groups are distribution groups whose membership is based on
...
-$FTE = Get-DynamicDistributionGroup "Full Time Employees"
+Get-DynamicDistributionGroupMember -Identity "Full Time Employees"
 ```
 
-```PowerShell
-Get-Recipient -RecipientPreviewFilter $FTE.RecipientFilter -OrganizationalUnit $FTE.RecipientContainer
-```
-
-This example displays the list of users and email addresses (more than 1000 mailboxes).
+This example displays the list of users and email addresses for the same group if it has more than 1000 mailboxes.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-dynamic-distribution-groups/view-group-members.md:45]:
```suggestion
This example displays the list of users and email addresses for the same group if it has more than 1,000 mailboxes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chrisda** [comment]
@dariomws, I knew that changes to DDG behavior was coming to ExO, but I didn't know the timreframe. Get-DynamicDistributionGroupMember is available to me in my 'normal" E5 tenant, so I added a new topic for it:

<https://github.com/MicrosoftDocs/office-docs-powershell/pull/8159>

**Comment by dariomws** [comment]
Thanks @chrisda!

**Pull request merged**